superset-frontend/src/dashboard/actions/nativeFilters.ts

```diff
@@ -49,6 +49,11 @@ export interface SetFilterConfigFail {
   type: typeof SET_FILTER_CONFIG_FAIL;
   filterConfig: FilterConfiguration;
 }
+export const SET_FILTER_SCOPES = 'SET_FILTER_SCOPES';
+export interface SetFilterScopes {
+  type: typeof SET_FILTER_SCOPES;
+  filterConfig: FilterConfiguration;
+}
```

Comment:
To avoid potential confusion with the scopes defined in metadata, it could be nice to call this (and related variables/functions) something more explicit, like `SET_IN_SCOPE_STATUS_OF_FILTERS`.

Reply:
Good point, done!